exercise_perseus_render/assets/src/module.js

```diff
@@ -1,6 +1,9 @@
 
 const ContentRendererModule = require('content_renderer_module');
 const ExerciseComponent = require('./vue/index');
+// Add window.btoa polyfill
+const btoa = require("btoa");
+global.btoa = window.btoa = btoa;
```

Comment:
Would probably be better to conditionally apply this, so that we don't overwrite preexisting btoa?:
`global.btoa = global.btoa ? btoa`